types/artifacts.d.ts

```diff
@@ -925,6 +926,19 @@ declare module Artifacts {
     /** The MIME type of the underlying image file. */
     mimeType?: string;
   }
+
+  interface Entity extends IEntity {
+    isUnrecognized?: boolean;
+  }
+
+  interface EntityClassification {
```

Comment:
Any way this can be moved into the file itself? Type dumping grounds make me sad, and since this isn't an artifact but a live function return value, seems better to have defined with the source?

Reply:
This is cross referenced across several audits and is there a cleaner way to share? `LH.Artifacts.*` is where all computed audits that require sharing is defined.